docs/kube-up.md

```diff
@@ -0,0 +1,22 @@
+# Production Quality Cluster on GCE
+This document provides the steps to bring up a production quality cluster on GCE with [`kube-up.sh`](https://kubernetes.io/docs/getting-started-guides/gce/).
+
+## Download CRI-Containerd Release Tarball
+To download release tarball, see [step 1](./installation.md#step-1-download-cri-containerd-release-tarball) in installation.md.
+
+Unpack release tarball to any directory, using `${CRI_CONTAINERD_PATH}` to indicate the directory in the doc:
+```shell
```

Comment:
console

Reply:
I'm not doing that intentionally. :)
We are not demonstrating the command result here, we want user to be able to copy and paste easily.

Just as what we do in README.md.

Reply:
how does using console vs shell make it harder to cut and paste?

Reply:
Yeah, console/shell makes no different. Will change to console then.

Reply:
Done.